internal/ibmcloud-janitor/resources/vpc_lbs.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type VPCLoadBalancer struct{}
+
+var lbTimeout = time.Minute * 10
+var lbWaitTime = time.Minute * 1
+
+// Cleans up the load balancers in a given region
+func (VPCLoadBalancer) cleanup(options *CleanupOptions) error {
+	resourceLogger := logrus.WithFields(logrus.Fields{"resource": options.Resource.Name})
+	resourceLogger.Info("Cleaning up the load balancers")
+	client, err := NewVPCClient(options)
+	if err != nil {
+		return errors.Wrap(err, "couldn't create VPC client")
+	}
+
+	var lbList map[string]bool
```

Comment:
can we rename it to denote that it contains list of deleted LBs?

Reply:
It's not initialised, need to use make.

Reply:
yes, fixed it.